# Add a control group to the manager

ControlGroup group
----